models/han/args.py

```diff
@@ -19,7 +19,6 @@ def get_args():
     parser.add_argument('--seed', type=int, default=3435)
     parser.add_argument('--dataset', type=str, default='SST-1', choices=['SST-1', 'SST-2', 'Reuters', 'AAPD', 'IMDB', 'Yelp2014'])
     parser.add_argument('--resume_snapshot', type=str, default=None)
-    parser.add_argument('--dev_every', type=int, default=30)
     parser.add_argument('--log_every', type=int, default=10)
```

Comment:
`--log-every`, `--resume-snapshot`?

Reply:
I guess --log-every could be moved to the common args file but resume-snapshot is actually dependent on the task. For doc classification, it's loading a model whereas for relevance transfer it is loading the previous run for all topics. I am leaving the latter as it is.

Reply:
Oh wait, the reason it's still here for HAN is because I haven't started working on HAN yet. I'm fixng up `reg_lstm` and `kim_cnn` first.